torchvision/transforms/functional_tensor.py

```diff
@@ -326,7 +325,9 @@ def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
     if not (-0.5 <= hue_factor <= 0.5):
         raise ValueError('hue_factor ({}) is not in [-0.5, 0.5].'.format(hue_factor))
 
-    if not (isinstance(img, torch.Tensor) and _is_tensor_a_torch_image(img)):
+    _assert_image_tensor(img)
+
+    if not (isinstance(img, torch.Tensor)):
```

Comment:
We probably want to move this check before the `_assert_image_tensor()` because in the method you already assume the input is a Tensor.